python/ray/monitor.py

```diff
@@ -364,6 +376,7 @@ def run(self):
         # Something went wrong, so push an error to all drivers.
         redis_client = ray._private.services.create_redis_client(
             args.redis_address, password=args.redis_password)
+        import ray.utils
```

Comment:
?

Reply:
I import it here only since it is needed when an exception happens.

Reply:
Are you suggesting to remove the entire:
```
        # Something went wrong, so push an error to all drivers.
        redis_client = ray._private.services.create_redis_client(
            args.redis_address, password=args.redis_password)
        import ray.utils
        traceback_str = ray.utils.format_error_message(traceback.format_exc())
        message = ("The monitor failed with the "
                   f"following error:\n{traceback_str}")
        ray.utils.push_error_to_driver_through_redis(
            redis_client, ray_constants.MONITOR_DIED_ERROR, message)
```
it does not seem like `push_error_to_driver_through_redis` is a no op in ray.utils?

Reply:
I synced with alex offline, he wanted to remove:
`traceback_str = ray.utils.format_error_message(traceback.format_exc())`